pom.xml

```diff
@@ -64,13 +64,16 @@
 		<liberty.remote.artifactId>arquillian-liberty-remote-jakarta</liberty.remote.artifactId>
 
 		<junit.version>4.13.2</junit.version>
+		<junit5.version>5.9.0</junit5.version>
```

Comment:
There is a `5.9.1` available.

Reply:
I used 5.9.0 because that is the provided dependency version for `arquillian-junit5-container` I figured it would be easier to keep these the same, but I could update this to 5.9.1 if you prefer.
REF: https://mvnrepository.com/artifact/org.jboss.arquillian.junit5/arquillian-junit5-container/1.7.0.Alpha13